src/benchmarklib/cli_config_parser.cpp

```diff
@@ -57,28 +50,36 @@ BenchmarkConfig CLIConfigParser::parse_basic_options_json_config(const nlohmann:
             << std::endl;
 
   const auto clients = json_config.value("clients", default_config.clients);
-  std::cout << "- " + std::to_string(clients) + " simulated clients are scheduling queries in parallel" << std::endl;
+  std::cout << "- " + std::to_string(clients) + " simulated clients are scheduling items in parallel" << std::endl;
 
   if (cores != default_config.cores || clients != default_config.clients) {
     if (!enable_scheduler) {
       PerformanceWarning("'--cores' or '--clients' specified but ignored, because '--scheduler' is false")
     }
   }
 
+  if (enable_scheduler && clients == 1) {
```

Comment:
what happens if someone calls the benchmark with `--clients 0`? or even `--clients -1`?

Reply:
Added assert